src/pages/trx-details/index.js

```diff
@@ -0,0 +1,273 @@
+import { useState, useContext } from 'react'
+import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
+import { RequestsContext } from 'src/providers/RequestsContext'
+import Table from 'src/components/Shared/Table'
+import { useResourceQuery } from 'src/hooks/resource'
+import { Grid, Button } from '@mui/material'
+import { ResourceIds } from 'src/resources/ResourceIds'
+import { useForm } from 'src/hooks/form'
+import { VertLayout } from 'src/components/Shared/Layouts/VertLayout'
+import { Fixed } from 'src/components/Shared/Layouts/Fixed'
+import { Grow } from 'src/components/Shared/Layouts/Grow'
+import { ResourceLookup } from 'src/components/Shared/ResourceLookup'
+import FormShell from 'src/components/Shared/FormShell'
+import CustomDatePicker from 'src/components/Inputs/CustomDatePicker'
+import { SystemRepository } from 'src/repositories/SystemRepository'
+import { DataSets } from 'src/resources/DataSets'
+import CustomTextField from 'src/components/Inputs/CustomTextField'
+import { useWindow } from 'src/windows'
+import DataForm from './form/DataForm'
+
+const TrxDetails = () => {
+  const [data, setData] = useState([])
+  const { getRequest } = useContext(RequestsContext)
+  const { stack } = useWindow()
+
+  const formatDate = date => {
```

Comment:
check if you should add it in date-helper file or if there is already one

Reply:
no